src/main/webapp/app/admin/user-management/user-management.component.html

```diff
@@ -176,6 +176,9 @@ <h2>
                                 <fa-icon [icon]="faWrench"></fa-icon>
                                 <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                             </a>
+                            <a class="btn btn-success btn-sm" (click)="ldapSync(user.login!)">
+                                <span class="d-none d-md-inline">LDAP</span>
```

Comment:
I suggest to use translations here and call it `Sync from LDAP` in English

Reply:
Changes accepted. In German, I used the translation "von LDAP synchronisieren"